src/content/en/fundamentals/performance/rail.md

```diff
@@ -1,171 +1,350 @@
-
-<table>
-  <thead>
-      <th>RAIL Step</th>
-      <th>Key Metric</th>
-      <th>User Actions</th>
-  </thead>
-  <tbody>
-    <tr>
-      <td data-th="RAIL Step"><strong>Response</strong></td>
-      <td data-th="Key Metric">Input latency (from tap to paint) < 100ms.</td>
-      <td data-th="User Test">User taps a button (for example, opening navigation).</td>
-    </tr>
-    <tr>
-      <td data-th="RAIL Step"><strong>Animation</strong></td>
-      <td data-th="Key Metric">Each frame's work (from JS to paint) completes < 16ms.</td>
-      <td data-th="User Test">User scrolls the page, drags a finger (to open
-        a menu, for example), or sees an animation. When dragging, the app's
-        response is bound to the finger position, such as pulling to refresh,
-        or swiping a carousel. This metric applies only to the continuous
-        phase of drags, not the start.
-      </td>
-    </tr>
-    <tr>
-      <td data-th="RAIL Step"><strong>Idle</strong></td>
-      <td data-th="Key Metric">Main thread JS work chunked no larger than 50ms.</td>
-      <td data-th="User Test">User isn't interacting with the page, but main thread should be available enough to handle the next user input.</td>
-    </tr>
-    <tr>
-      <td data-th="RAIL Step"><strong>Load</strong></td>
-      <td data-th="Key Metric">Page considered ready to use in 1000ms.</td>
-      <td data-th="User Test">User loads the page and sees the critical path content.</td>
-    </tr>
-  </tbody>
-</table> 
-
-
+**Goal**: Maximize idle time to increase the odds that the page responds to user input within
+50ms.
+
+**Guidelines**: 
+
+* Use idle time to complete deferred work. For example, for the initial page load, load as little
+  data as possible, then use idle time to load the rest.
+* Perform work during idle time in 50ms or less. Any longer, and you risk interfering with the
+  app's ability to respond to user input within 50ms.
+* If a user interacts with a page during idle time work, the user interaction should always
+  take the highest priority and interrupt the idle time work.
+
+## Load: deliver content and become interactive in under 5 seconds {: #load }
+
+When pages load slowly, user attention wanders, and users perceive the task as broken. Sites
+that load quickly have longer average sessions, lower bounce rates, and higher ad viewability.
+See [The Need For Mobile Speed: How Mobile Latency Impacts Publisher Revenue][NEED4SPEED].
+
+[NEED4SPEED]: https://www.doubleclickbygoogle.com/articles/mobile-speed-matters/
+
+**Goals**: 
+
+* For first loads, load the page and be [interactive][tti] in 5 seconds or less on mid-range
+  mobile devices with slow 3G connections. See [Can You Afford It? Real-World Web Performance
+  Budgets][Budgets].
+* For subsequent loads, load the page in under 2 seconds.
```

Comment:
Aren't these "goals" actually guidelines? The metric for a "fast load" is relative to the status quo. 1s load time for mobile devices is unrealistic. But if status quo conditions change over time, then 3s (for example) may become realistic. Therefore we'll need to change the "goal", therefore it's not "timeless"

I'm just wondering if there's a way to phrase this that makes it truly timeless guidance

Reply:
Like maybe "be aware of the typical load performance that your users experience, based on their devices and network connections, then aim for a fast loading experience relative to those constraints"

Reply:
Perhaps, "Optimize for fast loading performance relative to the devices and network conditions your users use to access your site"? I agree that for the guidance to be truly timeless it shouldn't be tied to a specific point-in-time constraint like TTI in 5s (we may change this in a year or two). 

What I'd like us for the guideline of TTI in <5s on median range mobile devices over slow 3G to be something captured lower down in the doc